.github/workflows/windows-vcpkg.yml

```
@@ -140,6 +140,13 @@ jobs:
            cd _build
            ctest -C ${{ matrix.buildtype }} --output-on-failure -L "unit|end-to-end" -LE ortools
 
+    - name: Run kirchhoff-constraint-tests
+      shell: bash
+      run: |
+           tar xvf src/tests/kirchhoff-cbuilder/reference.tgz
```

This task fails. Try this ?
```suggestion
           tar xvf src\tests\kirchhoff-cbuilder\reference.tgz
```